modules/nf-core/manta/somatic/main.nf

```diff
@@ -29,7 +29,7 @@ process MANTA_SOMATIC {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def options_manta = target_bed ? "--exome --callRegions $target_bed" : ""
+    def options_manta = target_bed ? "--callRegions $target_bed" : ""
```

Comment:
🤨 this was already merged in?: https://github.com/nf-core/sarek/pull/998/files

Reply:
Apparently